sunpy/util/decorators.py

```diff
@@ -182,3 +183,28 @@ def deprecate(obj, message=message, name=name, alternative=alternative):
         return deprecate(message)
 
     return deprecate
+
+
+class add_common_docstring(object):
+    """
+    A function decorator that will append and/or prepend an addendum
+    to the docstring of the target function.
+
+    If a dictionary(d) is provided as parameter then format by .format().
+    """
+    def __init__(self, append='', prepend='', d={}):
```

Comment:
If this isn't changed to `**kwargs` it should default to `None` not `{}` and then have a check for `None` in the body of the function. See [this](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments) for more info.